templates/layouts/components/media-gallery.php

```diff
@@ -0,0 +1,33 @@
+<div class="media-gallery">
+    <h5 class="text-area">Instagram</h5>
+    <div class="gallery-items-landscape">
+                <div class="gallery-items">
+                    <div class="gallery-items-images">
+                        <img width="150" height="150" src="http://wp.efforttech.net/newwp/sigma/wp-content/uploads/2019/01/image-1-5-150x150.jpg" class="attachment-thumbnail size-thumbnail" alt="" srcset="http://wp.efforttech.net/newwp/sigma/wp-content/uploads/2019/01/image-1-5-150x150.jpg 150w, http://wp.efforttech.net/newwp/sigma/wp-content/uploads/2019/01/image-1-5-300x300.jpg 300w, http://wp.efforttech.net/newwp/sigma/wp-content/uploads/2019/01/image-1-5-100x100.jpg 100w, http://wp.efforttech.net/newwp/sigma/wp-content/uploads/2019/01/image-1-5-60x60.jpg 60w, http://wp.efforttech.net/newwp/sigma/wp-content/uploads/2019/01/image-1-5.jpg 550w" sizes="(max-width: 150px) 100vw, 150px">
+                    </div>
```

Comment:
Please download these images and keep it in the images folder and add the image path here.

Reply:
i have downloaded the images but with direct path its not reflecting. ill check again

Reply:
It Should if you give the correct path... 
Remember this file is included in index.php so consider you are in root. 
So path should be something like:
`src="assets/images/xyz.png"`